.github/workflows/release-plugin.yml

```
@@ -1,33 +1,233 @@
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email 'github-actions[bot]@users.noreply.github.com'
+
+      - name: Checkout the specific branch/ref
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ env.SOURCE_REF }}
+          fetch-depth: 0
+
+      - name: Tag the release
+        id: tag
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:11:32: Double quote to prevent globbing and word splitting

(shellcheck)